### PR TITLE
Include watdiv ingest time other databases

### LINKED ISSUE #751: Bench includes ingest times for watdiv for the comparison DBs in the daily reports

### PR DESCRIPTION
**Pull request by danmason**
Resolves #751

### CODE REVIEW

**Comment by danmason** [comment]
This failed on AWS - it couldn't do `(json/write-str)` on duration. Will convert it to a string.

File: crux-bench/src/crux/bench/watdiv_datomic.clj [hunk 1]
```diff
@@ -351,11 +351,12 @@
...
 
 (defn -main []
-  (let [output-file (io/file "datomic-results.edn")]
-    (bench/save-to-file output-file
-                        (->> (run-watdiv-bench {:test-count 100})
-                             (filter :query-idx)
-                             (sort-by :query-idx)))
+  (let [output-file (io/file "datomic-results.edn")
+        watdiv-results (run-watdiv-bench {:test-count 100})]
+    (bench/save-to-file output-file (conj (->> (rest watdiv-results)
```
Comment (jarohen) [inline crux-bench/src/crux/bench/watdiv_datomic.clj]: nit: this is correct, but when I was reading it I had to double-check whether you wanted to put the `(first watdiv-results)` on the end or the beginning of this seq. let's take away the ambiguity with `(cons (first ...) (->> (rest ...) ...))`?

File: crux-bench/src/crux/bench/watdiv_crux.clj [hunk 2]
```diff
@@ -7,28 +7,41 @@
...
-
-(defn parse-results [resource]
-  (with-open [rdr (io/reader resource)]
-    (->> (line-seq rdr)
-         (map read-string)
-         (filter :result-count)
-         (map (juxt :query-idx identity))
-         (into {}))))
+                      :node-size-bytes (bench/node-size-in-bytes node)
+                      :neo4j-time-taken (get-in parsed-db-results [:neo4j :ingest])
```
Comment (jarohen) [inline crux-bench/src/crux/bench/watdiv_crux.clj]: units, maybe?

Comment (jarohen) [reply crux-bench/src/crux/bench/watdiv_crux.clj]: oh, I see, these are Durations. given we're still in raw-data territory here, (indeed, this is what's getting sent to CloudWatch?) let's try migrate the conversion to strings until we're rendering the results?

**Review by jarohen — CHANGES_REQUESTED** [review]
cheers Dan :) few little bits to update but nothing major, nicely done

File: crux-bench/src/crux/bench/watdiv_crux.clj (hunk 2 above)
Comment (danmason) [reply crux-bench/src/crux/bench/watdiv_crux.clj]: In which case, should move the conversion to a Duration in general.

**Pull request merged**